src/types/ErrorMessage.ts

```diff
@@ -0,0 +1,4 @@
+export enum ErrorMessage {
+  peopleLoadingError = 'Something went wrong',
+  noPeopleOnServer = 'There are no people on the server',
```

Comment:
Names of enum values are usually written either in all capital letters or starting with a capital letter
```suggestion
  PeopleLoadingError = 'Something went wrong',
  NoPeopleOnServer = 'There are no people on the server',
```